enterprise-suite/scripts/build-operator.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+console_version="1.2.0"
+docker_registry="lightbend-docker-commercial-registry.bintray.io"
+image_name="enterprise-suite/console-operator"
+full_docker_name="${docker_registry}/${image_name}:${console_version}"
+
+operator-sdk new console-operator --type=helm --kind=LBConsole \
```

Comment:
Maybe just `Console` if the api-version has `lightbend.com` already.

Reply:
Changed